modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -0,0 +1,354 @@
+[[clients-slemicro]]
+= Registering {slemicro} Clients
+
+This section contains information about registering clients running these {slemicro} operating systems:
+
+* {slemicro}{nbsp}5.0 {x86_64}
+* {slemicro}{nbsp}5.0 {arm64}
+
+ifeval::[{uyuni-content} == true]
+* {opensuse}{nbsp}{microos}
+endif::[]
+
+
+[WARNING]
+====
+This section is provided as a technology preview for testing purposes.
+Do not use this feature on production systems.
+====
+
+
+////
+Probably don't need to include this now. --LKB 2021-04-15
```

Comment:
Right, check before yo umerge, because the repos should show up at the Setup Wizard already.

Reply:
Yeah, I could see it in refhead last week.